test/e2e/runner/perturb.go

```diff
@@ -64,6 +64,10 @@ func PerturbNode(ctx context.Context, node *e2e.Node, perturbation e2e.Perturbat
 		}
 		if node.PersistInterval == 0 {
 			timeout *= 5
+		} else {
```

Comment:
What is the rationale of this if?

I assume that when `PersistInterval == 0`, the application has no state so it should restart from genesis, is that right?

Reply:
to be honest, just looking at the PersistInterval field, I only see it used in validation and perturb logic but in `if` clauses, so I think this field might not have logic that drives the state persistence. We can revisit it in the future, my PR just adds more time to the timeout in this case.